packages/sdk/src/utils/deprecate.js

```diff
@@ -1,29 +1,20 @@
 import * as logger from "@onflow/util-logger"
 
 const buildWarningMessage = ({name, transitionsPath}) => {
-  console.warn(
-    `
-    %cFCL/SDK Deprecation Notice
-    ============================
-    The ${name} builder has been deprecated and will be removed in future versions of the Flow JS-SDK/FCL.
-    You can learn more (including a guide on common transition paths) here: ${transitionsPath}
-    ============================
-  `,
-    "font-weight:bold;font-family:monospace;"
-  )
+  logger.log.deprecate({
+    pkg: "FCL/SDK",
+    subject: `The ${name} builder`,
+    transition: transitionsPath,
+  })
 }
 
 const buildErrorMessage = ({name, transitionsPath}) => {
-  console.error(
-    `
-    %cFCL/SDK Deprecation Notice
-    ============================
-    The ${name} builder has been removed from the Flow JS-SDK/FCL.
-    You can learn more (including a guide on common transition paths) here: ${transitionsPath}
-    ============================
-  `,
-    "font-weight:bold;font-family:monospace;"
-  )
+  logger.log.deprecate({
+    pkg: "FCL/SDK",
+    message: `The ${name} builder has been removed from the Flow JS-SDK/FCL.`,
+    transition: transitionsPath,
+    level: logger.LEVELS.error,
+  })
 }
 
 const warn = deprecated => buildWarningMessage(deprecated)
```

Comment:
Lookin good. Maybe we just rip this out
https://github.com/onflow/fcl-js/blob/jribbink/standardize-deprecation/packages/sdk/src/utils/deprecate.js
buildWarn, buildError out and replace with your new `deprecate`
https://github.com/onflow/fcl-js/blob/jribbink/standardize-deprecation/packages/sdk/src/sdk.js

Reply:
Alright.  I've tossed both the utils folders for fcl & sdk as all they had in them were deprecation function which have been since superseded